webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -383,18 +387,47 @@ const UtterancesTable: React.FC<Props> = ({
         />
         <Button
           className={classes.exportButton}
-          onClick={() =>
-            downloadDatasetSplit({
-              jobId,
-              datasetSplitName,
-              ...filters,
-              ...pipeline,
-            })
+          onClick={(event: React.MouseEvent<HTMLButtonElement>) =>
+            setAnchorEl(event.currentTarget)
           }
```

Comment:
Since you define the function in line, you don't need to specify the type of `event`.
```suggestion
          onClick={(event) => setAnchorEl(event.currentTarget)}
```

Reply:
Okay.  Right. Done.